running make gives `sed: 1: "CITATION.cff": invalid command code C`

I added a new template and ran `make`. I get:

```
V=$(poetry run python -c "import ontogpt;print('.'.join((ontogpt.__version__).split('.', 3)[:3]))") ; \
	sed -i '/^version:/c\version: '"$V"'' CITATION.cff
sed: 1: "CITATION.cff": invalid command code C
make: *** [update_citation] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -21,10 +21,8 @@ unit-test:
 integration-test:
 	$(RUN) python -m unittest
 
-
-update_citation: CITATION.cff
-	V=$$($(RUN) python -c "import ontogpt;print('.'.join((ontogpt.__version__).split('.', 3)[:3]))") ; \
-	sed -i '/^version:/c\version: '"$$V"'' $<
+get_version:
+	$(RUN) python -c "import ontogpt;print('.'.join((ontogpt.__version__).split('.', 3)[:3]))"
 
 $(TEMPLATE_DIR)/%.py: src/$(PACKAGE)/templates/%.yaml
 	$(RUN) gen-pydantic $< > $@.tmp && mv $@.tmp $@

--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ EVAL_DIR = src/$(PACKAGE)/evaluation
 TEMPLATES = core gocam mendelian_disease biological_process treatment environmental_sample metagenome_study reaction recipe ontology_class metabolic_process drug ctd halo gene_description_term
 ENTRY_CLASSES = recipe.Recipe gocam.GoCamAnnotations reaction.ReactionDocument ctd.ChemicalToDiseaseDocument
 
-all: all_pydantic all_projects update_citation
+all: all_pydantic all_projects
 
 all_pydantic: $(patsubst %, $(TEMPLATE_DIR)/%.py, $(TEMPLATES))
 all_projects: $(patsubst %, projects/%, $(TEMPLATES))

